src/pages/AdminDashboard/roomsActions.ts

```diff
@@ -0,0 +1,65 @@
+import { RoomsActionsTypes, SET_ROOMS } from './roomsActionTypes'
+import { Api } from '../../services/api'
+import { showErrorMessage } from '../../components/App/Snackbar/snackbarActions'
+import {
+    hideBackdrop,
+    showBackdrop,
+} from '../../components/App/Backdrop/backdropActions'
+
+export const setRooms = (rooms: any): RoomsActionsTypes => ({
+    type: SET_ROOMS,
+    payload: {
+        rooms,
+    },
+})
+
+export const getRooms = (t) => async (dispatch, getState) => {
+    const { user: userState } = getState()
+    const { token } = userState.user
+    const api = new Api(token)
+
+    try {
+        const rooms = await api.getRooms()
+        dispatch(setRooms(rooms))
+    } catch (err) {
+        dispatch(showErrorMessage(t('errors.rooms-fetch')))
+    }
+}
+
+export const createRoom = (t, roomName, hostName, destinations) => async (
```

Comment:
It's just my opinion. I prefer to give literal object when function have many parameters :p

Reply:
I agree ;) I just focused on functionality for time reasons. It will be refactored ;)